bochs/configure.ac

```diff
@@ -1252,6 +1252,23 @@ if test "$support_cet" -gt 0 -a "$use_x86_64" = 0; then
   AC_MSG_ERROR([Control Flow Enforcement Technology feature emulation require x86-64 enabled])
 fi
 
+AC_MSG_CHECKING(for KeyLocker support)
+AC_ARG_ENABLE(3dnow,
```

Comment:
Is `3dnow` related to this change? If not, looks like mistake.

Reply:
Thanks, will fix !